run_unittests.py

```diff
@@ -2833,6 +2959,11 @@ def test_using_recent_feature(self):
                                       ".*WARNING.*Project targetting.*but.*",
                                       meson_version='>= 0.47.0')
 
+    def test_feature_invalid_version(self):
+        os.environ['MESON_UNIT_TEST'] = '1'
+        self.assertMesonRaises("exception(mode: 'red')",
+                                "Invalid version string", meson_version=">0.44.0")
```

Comment:
__[Flake8]__

[E127] continuation line over-indented for visual indent


<!-- Comment by [Sider](https://sider.review) -->